src/mavis/annotate/file_io.py

```diff
@@ -35,26 +41,42 @@ def load_masking_regions(*filepaths: str) -> Dict[str, List[BioInterval]]:
 
         #chr    start       end         name
         chr20   25600000    27500000    centromere
-
     Args:
-        filepath: path to the input tab-delimited file
+        filepath: path to standard MAVIS format file
     Returns:
-        a dictionary keyed by chromosome name with values of lists of regions on the chromosome
+        a dictionary with {(bp1_chr,bp2_chr):{BreakpointPair}}
```

Comment:
This doesn't match the return type annotation you have above

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).